src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_kds.c

```diff
@@ -325,18 +325,25 @@ static inline void read_ert_stat(struct kds_command *xcmd)
 	 */
 
 	/* New KDS handle FPGA CU statistic on host not ERT */
-	if (ecmd->data[0] != 0x51a10000)
+	if ((ecmd->data[0] != ERT_CUSTAT_VERSION_0) && 
+			(ecmd->data[0] != ERT_CUSTAT_VERSION_1))
 		return;
 
 	/* Only need PS kernel info, which is after FPGA CUs */
 	mutex_lock(&kds->scu_mgmt.lock);
 	/* Skip header and FPGA CU stats. off_idx points to PS kernel stats */
 	off_idx = 4 + num_cu;
```

Comment:
What is so magical about the value 4.

Reply:
Fixed. Thanks for this suggestion. Its looks good now and more readable.